tools/Makefile.unix

```diff
@@ -437,9 +435,7 @@ ifeq ($(CONFIG_BUILD_2PASS),y)
 	$(Q) $(MAKE) -C $(CONFIG_PASS1_BUILDIR) TOPDIR="$(TOPDIR)" LINKLIBS="$(LINKLIBS)" USERLIBS="$(USERLIBS)" "$(CONFIG_PASS1_TARGET)"
 endif
 
-pass2deps: pass2dep $(NUTTXLIBS)
```

Comment:
`export` still depends on `pass2deps`.
Similarly in Makefile.win

Reply:
Just update it. Thanks point out.